PyInstaller/hooks/pre_safe_import_module/hook-six.moves.py

```diff
@@ -27,24 +27,20 @@ def pre_safe_import_module(api):
     tkinter_tix` to `import tkinter.tix`). For each such mapping, this hook adds a corresponding module alias to the
     graph allowing PyInstaller to translate the former to the latter.
     """
-    # Dictionary from conventional module names to "six.moves" attribute names (e.g., from `tkinter.tix` to
-    # `six.moves.tkinter_tix`).
-    real_to_six_module_name = eval_statement(
-        """
+    @isolated.call
+    def real_to_six_module_name():
+        """Generate a dictionary from conventional module names to "six.moves" attribute names (e.g., from `tkinter.tix`
+         to `six.moves.tkinter_tix`)."""
```

Comment:
With the reformatting commit, the docstring descriptions for functions are methods were changed to 
```
"""
String...
"""
```
in an attempt to keep them consistent across the codebase. I'd prefer if we keep it that way, but that's obviously not a dealbreaker.

The same comment applies to all such strings within this PR.

Reply:
Hmm, this will be a fun habit to break...